clients/snowflake/staging.go

```diff
@@ -43,7 +42,7 @@ func replaceExceededValues(colVal string, kindDetails typing.KindDetails) string
 
 // castColValStaging - takes `colVal` any and `colKind` typing.Column and converts the value into a string value
```

Comment:
Update this comment to say `typing.KindDetails` instead of `typing.Column`? (or just remove the types from the comment and let the function be the source of truth)

Reply:
sg